torch/export/_trace.py

```diff
@@ -575,7 +576,7 @@ def make_argument_spec(i, node) -> ArgumentSpec:
         if i < len(graph_signature.input_tokens):
             # TODO: We should be checking for a different type, once we add a new type
             return TokenArgument(name=node.name)
-        elif isinstance(val, FakeTensor):
+        elif isinstance(val, FakeTensor) or is_sparse_any(val):
```

Comment:
Why sparse tensors do not end up being fake tensors here?

Reply:
same reason as for the test, right? direct argument vs. internal node?